src/video/v9990/V9990.hh

```diff
@@ -454,6 +454,25 @@ private:
 		RESERVED_3
 	};
 
+	std::string portName[16] = {
+		"VRAM_DATA",
+		"PALETTE_DATA",
+		"COMMAND_DATA",
+		"REGISTER_DATA",
+		"REGISTER_SELECT",
+		"STATUS",
+		"INTERRUPT_FLAG",
+		"SYSTEM_CONTROL",
+		"KANJI_ROM_0",
+		"KANJI_ROM_1",
+		"KANJI_ROM_2",
+		"KANJI_ROM_3",
+		"RESERVED_0",
+		"RESERVED_1",
+		"RESERVED_2",
+		"RESERVED_3"
+	};
+
```

Comment:
This can be moved from the .hh to the .cc file (until we also need it in some other place than V9990.cc).
I'd also suggest to make the following tweaks:
* Use `std::string_view` instead of `std::string`. This is more light-weight. E.g. it allows for compile-time initialization rather than run-time initialization.
* Declare this array `static` because (once moved to the .cc file) it should only be visible in one file.
* Declare it `constexpr` to indicate that the table won't change (`constexpr` implies `const`) and that we want it to be initialized at compile-time.
* We prefer to use `std::array` over plain C-arrays (for reasons I won't go into now).

In other words, declare this table as:
```
static constexpr std::array<std::string_view, 16> portName = {
    "VRAM_DATA",
    ...
};
```
I also wonder if, for users of this feature, it's clearer to show the port name (e.g. KANJI_ROM_0), the local port number (e.g. 8), or the full port number (e.g. 0x68). Or maybe some combination?

I'm assuming that you don't intentionally read/write from/to the wrong port. It may not even have been the intention to access the V9990 at all. And in that case having the full port number might be more useful than having the name of a V9990 port. (And if the user was programming the V9990, he/she probably knows the port numbers by heart.)

Reply:
Using port numbers only.